src/xrEngine/EngineAPI.cpp

```diff
@@ -130,13 +130,18 @@ void CEngineAPI::Initialize(void)
     InitializeRenderers();
 
     hGame = XRay::LoadModule("xrGame");
-    R_ASSERT2(hGame->IsLoaded(), "Game DLL raised exception during loading or there is no game DLL at all");
-
-    pCreate = (Factory_Create*)hGame->GetProcAddress("xrFactory_Create");
-    R_ASSERT(pCreate);
+    if (!hGame->IsLoaded())
+    {
+        Msg("! Game DLL raised exception during loading or there is no game DLL at all");
```

Comment:
This is not a good thing to allow xrGame loading to fail, consider adding `-no_game` key.